### PR TITLE
Pull Request for Issue1913: Implement Imaging carbon filter farm

### PR DESCRIPTION
**Pull request by helfrij**
- Created bioxasImagingCommon.pri file for Imaging-specific class files.
- Created BioXASImagingCarbonFilterFarm class, borrowing heavily from existing classes on Side/Main.
- Added instance to BioXASImagingBeamline, and view to BioXASImagingAppController.
- Setpoints for each of the actuator positions have been tested.

### CODE REVIEW

**Comment by helfrij** [comment]
Hey @XuetaoLiu-CLS, would you mind taking a look at this pull request?

**Comment by XuetaoLiu-CLS** [comment]
Looks good.

Time Estimates: 15 minutes

File: source/beamline/BioXAS/BioXASCarbonFilterFarm.cpp [hunk 1]
```diff
@@ -3,7 +3,7 @@
 BioXASCarbonFilterFarm::BioXASCarbonFilterFarm(const QString &deviceName, QObject *parent) :
 	BioXASCarbonFilterFarmControl(deviceName, parent)
 {
-	setFilter(new BioXASCarbonFilterFarmFilterControl(name()+"Filter", "mm", this));
+	setFilter(new BioXASCarbonFilterFarmFilterControl(name()+"Filter", "um", this));
```
Comment (dretrex) [inline source/beamline/BioXAS/BioXASCarbonFilterFarm.cpp]: Flagging use of + for string concatenation.

File: source/beamline/BioXAS/BioXASImagingCarbonFilterFarm.cpp [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	BioXASCarbonFilterFarm("BioXASImagingCarbonFilterFarm", parent)
+{
+	// Setup the upstream actuator.
+
+	BioXASCarbonFilterFarmActuator *actuator = new BioXASCarbonFilterFarmActuator(name()+"UpstreamActuator", this);
+
+	upstreamMotor_ = new CLSMAXvMotor("SMTR1607-5-I00-05", "SMTR1607-5-I00-05", "SMTR1607-5-I00-05", true, 0.05, 2.0, this);
+	actuator->setPositionControl(upstreamMotor_);
+
+	AMReadOnlyPVControl *motorStatus = new AMReadOnlyPVControl(name()+"UpstreamActuatorStatus", "PFIL1607-5-I10-01:InPosition", this);
```
Comment (dretrex) [inline source/beamline/BioXAS/BioXASImagingCarbonFilterFarm.cpp]: Here too.

**Comment by dretrex** [comment]
Overall, looks good!
Time Estimate: 5 minutes

**Pull request merged**